### ISSUE TITLE
abort in case make update runs as root

### ISSUE THREAD

**Issue by Strubbl** [wallabag/wallabag#3590]
### Issue details

reported by @francis-barban in the gitter chat

![grafik](https://user-images.githubusercontent.com/97055/36061082-5eaa570c-0e55-11e8-99ae-0e02c6dff26f.png)

detect when make update runs as root and abort in that case

### Environment

* wallabag version (or git revision) that exhibits the issue:
* How did you install wallabag? Via `git clone` or by downloading the package?
* Last wallabag version that did not exhibit the issue (if applicable):
* php version:
* OS:
* type of hosting (shared or dedicated):
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL):

### Steps to reproduce/test case

Please provide necessary steps for reproduction of this issue, or better the
reduced test case (without any external dependencies, if possible).

**Comment by francis-barban** [comment]
Not the real cause. Warning because security lost but runs well. Anyway you have to CHOWN your wallabag after cache clear command (which rebuild the cache files with root owner so no access for wallabag).

**Comment by Strubbl** [comment]
Yes, you are right. That's the reason I opened this second issue. This issue should respect the composer's warning. Having no git installed, is a different issue and handled by #3589
